src/commands/math/basicSymbols.js

```diff
@@ -414,8 +414,10 @@ var PlusMinus = P(BinaryOperator, function(_) {
 
   _.contactWeld = _.siblingCreated = _.siblingDeleted = function(opts, dir) {
     if (dir === R) return; // ignore if sibling only changed on the right
+    // If the left sibling is a binary operator or a separator (comma, full stop, semicolon, colon)
+    // consider the operator to be unary, otherwise binary
     this.jQ[0].className =
-      (!this[L] || this[L] instanceof BinaryOperator ? '' : 'mq-binary-operator');
+      (!this[L] || this[L] instanceof BinaryOperator || ',.;:'.indexOf(this[L].ctrlSeq) >= 0 ? '' : 'mq-binary-operator');
```

Comment:
I'm not sure about dot `.`, in math that's usually a decimal point, right? Like `1.0 - 1. - 1`, MathJax at least doesn't consider the minus in that case to be unary: http://mathb.in/55242